src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx

```diff
@@ -148,21 +146,14 @@ export const ConversationsList = ({
     <>
       <h2 className="visually-hidden">{t('conversationViewTooltip')}</h2>
 
-      <div>
-        <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name="conversation-view">
-          <ConnectionRequests
-            connectionRequests={connectRequests}
-            onConnectionRequestClick={onConnectionRequestClick}
-          />
-
-          <ConversationView
-            conversations={conversations}
-            conversationsFilter={conversationsFilter}
-            currentFolder={currentFolder}
-            getCommonConversationCellProps={getCommonConversationCellProps}
-          />
-        </ul>
-      </div>
+      <ConnectionRequests connectionRequests={connectRequests} onConnectionRequestClick={onConnectionRequestClick} />
+
+      <ConversationView
+        conversations={conversations}
+        conversationsFilter={conversationsFilter}
+        currentFolder={currentFolder}
+        getCommonConversationCellProps={getCommonConversationCellProps}
+      />
```

Comment:
If we pass conversations (filtered/unfiltered), we can prevent `ConversationView` from redundant code.

```
 const filteredConversations = currentFolder?.conversations().filter(conversationSearchFilter(conversationsFilter));
 const conversationsToDisplay = filteredConversations.length ? filteredConversations : conversation;
 
 // then
 
      <ConversationView
        conversations={conversationsToDisplay}
        getCommonConversationCellProps={getCommonConversationCellProps}
      />
 
```